lms/djangoapps/learner_dashboard/utils.py

```diff
@@ -44,8 +44,14 @@ def disclaimer_incomplete_fields_notification(self, request):
     delta = current_date - joined_date
 
     if delta.days > days_passed_threshold:
-        additional_fields = ['country', 'city', 'gender', 'year_of_birth', 'level_of_education', 'goals']
-        for field_name in additional_fields:
+        default_additional_fields = {
+            'fields': ['country', 'city', 'gender', 'year_of_birth', 'level_of_education', 'goals']
+        }
+        additional_fields = configuration_helpers.get_value(
+            'FIELDS_TO_CHECK_PROFILE_COMPLETION',
+            settings.FEATURES.get('FIELDS_TO_CHECK_PROFILE_COMPLETION', default_additional_fields)
+        )
+        for field_name in additional_fields['fields']:
```

Comment:
Any particular reason there is a `fields` key here? Also, shouldn't the default be set through configuration, rather than in the code like this?